### PR TITLE
Write review modal [WAIV-1134]

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/rewards/CampaignFooter/CampaignFooter.js [hunk 1]
```diff
@@ -217,13 +218,11 @@ class CampaignFooter extends React.Component {
...
       pendingFollowObject,
-      proposedWobj,
       requiredObjectPermlink,
-      requiredObjectName,
       intl,
-      proposition,
+      toggleModalDetails,
     } = this.props;
-
+    console.log('this.props', this.props);
```
Comment (milklim) [inline src/client/rewards/CampaignFooter/CampaignFooter.js]: remove

File: src/client/rewards/CampaignFooter/CampaignButtons.js [hunk 2]
```diff
@@ -210,17 +198,12 @@ export default class CampaignButtons extends React.Component {
           {this.renderPostPopoverMenu()}
         </div>
         <React.Fragment>
-          <LinkButton
-            block={false}
-            className="WriteReview-button"
-            to={`/editor?object=[${requiredObjectName}](${requiredObjectPermlink})&object=[${proposedObjectName}](${proposedObjectPermlink})&campaign=${propositionId}`}
-            type="primary"
-          >
+          <Button type="primary" onClick={() => toggleModalDetails({ value: true })}>
```
Comment (milklim) [inline src/client/rewards/CampaignFooter/CampaignButtons.js]: toggleModalDetails({ value: true }) - does it open modal? Better to add openModalDetails method to current component

File: src/client/rewards/Proposition/Proposition.js [hunk 3]
```diff
@@ -131,14 +133,16 @@ const Proposition = ({
       >
         {/*Temporary fix until changes on backend will be made*/}
         {/*{proposition.activation_permlink && assigned === true && !_.isEmpty(post) ? (*/}
-        {proposition.activation_permlink && assigned === true ? (
+        {/* changes braked reservation process, changes reverted */}
+        {proposition.activation_permlink && assigned === true && !_.isEmpty(post) ? (
```
Comment (milklim) [inline src/client/rewards/Proposition/Proposition.js]: import lodash methods separately

File: src/client/rewards/CampaignFooter/CampaignButtons.js (hunk 2 above)
Comment (konstantin1390) [reply src/client/rewards/CampaignFooter/CampaignButtons.js]: ++

File: src/client/rewards/CampaignFooter/CampaignFooter.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/rewards/CampaignFooter/CampaignFooter.js]: ++

File: src/client/rewards/Proposition/Proposition.js (hunk 3 above)
Comment (konstantin1390) [reply src/client/rewards/Proposition/Proposition.js]: ++

**Pull request merged**